### PR TITLE
Css: letters, messages

### PR DESCRIPTION
**Pull request by brdunfield**
- remove letter CSS
- rename existing message.css to email_sms_message.css. (it replaced email and sms css files previously)
- port message.scss

### CODE REVIEW

File: app/assets/stylesheets/tailwind/message.css [hunk 1]
```diff
@@ -1,50 +1,146 @@
 @layer components {
-  .sms-message-wrapper {
-    max-width: 464px;
-    word-wrap: break-word;
-    @apply relative w-full box-border p-gutterHalf rounded-lg whitespace-normal m-0 mb-gutter clear-both bg-gray-button border-gray-button border-solid border;
+  .message-name {
```
Comment (brdunfield) [inline app/assets/stylesheets/tailwind/message.css:2]: this diff is a bit messy due to the rename

File: app/assets/stylesheets/tailwind/message.css [hunk 2]
```diff
@@ -1,50 +1,146 @@
...
-    font-family: inherit !important;
-    @apply border-0 p-0;
+  .folder-heading a {
+    @apply inline-block align-top;
+  }
+  .folder-heading a.folder-heading-folder {
+    text-overflow: ellipsis;
+    background-position: 0px 4px;
+    @apply inline overflow-hidden bg-folderBluePng bg-folderBlue;
+  }
```
Comment (AntoineAugusti) [inline app/assets/stylesheets/tailwind/message.css:114]: Screen Shot 2020-12-01 at 11 32 31
I think that you ported the existing CSS but the vertical alignment seems off here

Comment (AntoineAugusti) [reply app/assets/stylesheets/tailwind/message.css:114]: Not sure why "Templates" is with an ellipsis here
Screen Shot 2020-12-01 at 11 34 32

**Review by AntoineAugusti — APPROVED** [review]
Tested a bit folders and templates on the review app. A few questions regarding folders

File: app/assets/stylesheets/tailwind/message.css (hunk 2 above)
Comment (brdunfield) [reply app/assets/stylesheets/tailwind/message.css:114]: Yes I noticed the folder alignment too - it's like that in the current CSS as well. But I can throw in a quick fix.

I will fix the Template ellipsis - thanks for catching!

Comment (brdunfield) [reply app/assets/stylesheets/tailwind/message.css:114]: The Template ellipsis is also in staging and prod, but i'll see if it's an easy fix

**Comment by brdunfield** [comment]
Folder alignment is trickier so I left it for now, but ellipsis was a quick fix

**Pull request merged**